src/Core/Services/Fido2AuthenticatorService.cs

```diff
@@ -0,0 +1,233 @@
+﻿using Bit.Core.Abstractions;
+using Bit.Core.Models.View;
+using Bit.Core.Enums;
+using Bit.Core.Models.Domain;
+using Bit.Core.Utilities.Fido2;
+
+namespace Bit.Core.Services
+{
+    public class Fido2AuthenticatorService : IFido2AuthenticatorService
+    {
+        private INativeLogService _logService;
```

Comment:
⚠️ `INativeLogService` is only for console output and most of the times only useful on DEBUG. Would you like the logs to be pushed to AppCenter? If so, use `ILogger`

Reply:
Removed logging until further notice